### DISCUSSION TITLE
Support for "absolute time" as calendar time

### DISCUSSION THREAD

**Discussion by sbenthall** [econ-ark/HARK#1674] [General]
See #717 

Support for calendar time, in order to better square HARK models with real data and predictions. From @llorracc 

> what I mean by “absolute” time is the ability to specify that a given moment in the simulation corresponds, say, to June 16, 2020.  The purpose would just be to make it easier to align model events with real events (e.g., the timing of the Covid-19 crisis, or the Great Recession).  Obviously, this can be done without an explicit measure of “absolute time” — we did it in the Covid-19 paper.  The question is whether it would be worthwhile to develop syntax or conventions for how to handle the alignment of a model with an actually measured historical sequence of events.

@mnwhite replies:

>  think we would want a *lot* more integration with data (read: do *something* with pandas) before we turn to this.  But I'm a partisan for keeping data separate from modeling, leaving the former to specific research projects.

**Comment by mnwhite** [comment]
I move to adopt Sebastian's terminology of "calendar time".  It's intuitive.
